api-reference/10 UI Components/GridBase/1 Configuration/rowDragging/onDragChange.md

```diff
@@ -22,25 +22,25 @@ Indicates if the dragged row is dropped inside another row. Available only for t
 #include common-ref-eventparam
 
 ##### field(e.fromComponent): dxSortable | dxDraggable
-The instance of the row's source UI component.
+The instance of the row's source UI component. Works only for dragging between components.
 
 ##### field(e.fromData): any
-Custom [data](/api-reference/10%20UI%20Components/GridBase/1%20Configuration/rowDragging/data.md '{basewidgetpath}/Configuration/rowDragging/#data') associated with the source UI component.
+Custom [data](/api-reference/10%20UI%20Components/GridBase/1%20Configuration/rowDragging/data.md '{basewidgetpath}/Configuration/rowDragging/#data') associated with the source UI component. Works only for dragging between components.
 
 ##### field(e.fromIndex): Number
 The row's original position. If the row is dragged from another UI component, the value is **-1**.
 
 ##### field(e.itemData): any
-The dragged row's data.
+The row's data before dragging.
```

Comment:
Just note it's not a comment needed to fix.

In the Grid and TreeList, the itemData before drag-n-drop equals the "itemData" after drag-n-drop.
Because in these components, the "itemData" doesn't contain any fields that will change during drag-n-drop interaction.

But I agree that mentioning that the "itemData" is data **before** drag-n-drop's interaction will be better.